config/vufind/searches.ini

```diff
@@ -254,10 +254,14 @@ WorkKeys = year
 ;
 ; Available modules recommended for use in the side area:
 ;
-; CatalogResults:[GET parameter]:[result limit]
+; CatalogResults:[GET parameter]:[result limit]:[hidden filters key]:[custom heading]
 ;       Display catalog search results matching the terms found in the specified
 ;       GET parameter (default = "lookfor"), limited to a specified number of
-;       matches (default = 5).  This is designed for use with non-catalog modules.
+;       matches (default = 5). This is designed for use with non-catalog modules.
+;       Hidden filters key, if defined, is a reference to hidden filters for these
+;       results, defined in the [AbstractSearchObjectHiddenFilters] section of the
+;       searches.ini file. The optional custom heading overrides the default for this
```

Comment:
I know it's weird to refer to "the searches.ini" file within itself.  But as the earlier note says CatalogResults is more likely to be used in another config file, while the [AbstractSearchObjectHiddenFilters] would still be defined here.

Reply:
I don't have a problem with referring to searches.ini here, though I guess if you wanted to avoid that you could say something more technical about using the search configuration file configured in the Options class (since that's technically more accurate if somebody has overridden that for some reason...).

Reply:
Yeah that's more accurate but I think even more confusing. :)  I'd rather leave it as is if you are ok with it.